src/providers/validationProvider.ts

```diff
@@ -54,10 +54,10 @@ export async function doValidate(
         : settings.validation.lint.path;
       const lintAvailability =
         await commandRunner.getExecutablePath(lintExecutable);
-      console.debug("Path for lint: ", lintAvailability);
+      connection?.console.log(`Path for lint: ${lintAvailability}`);
 
       if (lintAvailability) {
-        console.debug("Validating using ansible-lint");
+        connection?.console.log("Validating using ansible-lint");
```

Comment:
The decision to change `debug` to `log` was made because `debug` is not implemented on connection (type RemoteConsole) thus npm won't compile when this is changed. 

```log
src/providers/validationProvider.ts:57:27 - error TS2339: Property 'debug' does not exist on type 'RemoteConsole & _'.

57       connection?.console.debug("Path for lint: ", lintAvailability);
                             ~~~~~

src/providers/validationProvider.ts:60:29 - error TS2339: Property 'debug' does not exist on type 'RemoteConsole & _'.

60         connection?.console.debug("Validating using ansible-lint");
                               ~~~~~

src/providers/validationProvider.ts:71:27 - error TS2339: Property 'debug' does not exist on type 'RemoteConsole & _'.

71       connection?.console.debug("Validating using ansible syntax-check");
                             ~~~~~

src/providers/validationProvider.ts:74:29 - error TS2339: Property 'debug' does not exist on type 'RemoteConsole & _'.

74         connection?.console.debug("playbook file");
                               ~~~~~

src/providers/validationProvider.ts:78:29 - error TS2339: Property 'debug' does not exist on type 'RemoteConsole & _'.

78         connection?.console.debug("non-playbook file");
                               ~~~~~


Found 5 errors in the same file, starting at: src/providers/validationProvider.ts:57
```